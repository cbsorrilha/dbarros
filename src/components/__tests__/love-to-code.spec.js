import React from "react"
import renderer from "react-test-renderer"
import LoveToCode from '../love-to-code';


describe('Snapshot testing the LoveToCode', () => {
    it("renders correctly", () => {
        const props = {}
        const tree = renderer.create(<LoveToCode {...props} />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
