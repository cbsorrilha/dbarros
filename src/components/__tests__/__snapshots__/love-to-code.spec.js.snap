// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing the LoveToCode renders correctly 1`] = `
<div
  className="sc-AykKH igdBGH"
>
  <pre>
    <span
      className="sc-AykKC juMySS"
    >
      if
    </span>
    <span
      className="sc-AykKD eICLKN"
    >
      (
    </span>
    <span
      className="sc-AykKE IkwsY"
    >
      true
    </span>
    <span
      className="sc-AykKD eICLKN"
    >
      )
    </span>
    <span
      className="sc-AykKF fTPbnk"
    >
      {
    </span>
     
    <span
      className="sc-AykKG jTNUeG"
    >
      loveToCode
    </span>
    <span
      className="sc-AykKD eICLKN"
    >
      (
    </span>
    <span
      className="sc-AykKD eICLKN"
    >
      )
    </span>
    <span
      className="sc-AykKF fTPbnk"
    >
      ;
    </span>
     
    <span
      className="sc-AykKF fTPbnk"
    >
      }
    </span>
  </pre>
</div>
`;
